python/lsst/obs/base/cameraMapper.py

```diff
@@ -1073,5 +1069,19 @@ def exposureFromImage(image, logger=None):
     else:
         # Image
         exposure = afwImage.makeExposure(afwImage.makeMaskedImage(image))
+    #
+    # set VisitInfo if we can
+    #
+    if exposure.getInfo().getVisitInfo() is None:
+        if metadata is not None:
+            if mapper is None:
+                if not logger:
+                    logger = lsstLog.Log.getDefaultLogger()
```

Comment:
A different pattern is used above (`logger = lsstLog.Log.getLogger("obs.base.cameraMapper")`). Should they be the same?
I thought maybe the two should be combined, but then I thought that would probably mean making a logger every time this function is called, and maybe we don't want to do that.